src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/AddInstruction.cs

```diff
@@ -29,14 +31,14 @@ public override int Run(InterpretedFrame frame)
                 }
                 else
                 {
-                    frame.Data[frame.StackIndex - 2] = ScriptingRuntimeHelpers.Int32ToObject(unchecked((int)l + (int)r));
+                    frame.Data[frame.StackIndex - 2] = unchecked((short)((short)l + (short)r));
                 }
                 frame.StackIndex--;
-                return +1;
+                return 1;
```

Comment:
See https://github.com/dotnet/corefx/pull/14180 for a PR that makes the `+1` to `1` change across all instructions (also non-generated ones), thus reducing the noise here.